### PR TITLE
Updated LAA acronyms

### PR DESCRIPTION
**Pull request by carolinapizatto**
Amended and added new acronyms that are relevant to those of us working on the legal aid agency. 

A list of all acronyms that have been updated can be found on this link: https://dsdmoj.atlassian.net/wiki/spaces/LAA/pages/923992084/Acronyms

### CODE REVIEW

**Review by jakemulley — CHANGES_REQUESTED** [review]

File: README.md [hunk 1]
```diff
@@ -87,16 +88,17 @@ A better mechanism to enable people without github accounts to suggest changes w
...
 | CBO | crime billing online | | |
 | CCBC | County Court Bulk Centre | http://www.justice.gov.uk/courts/northampton-bulk-centre/ccbc | Set up by HM Courts & Tribunals Service specifically to deal with straightforward debt collection work which, in the main, is undefended. In Northampton. |
-| CCCD | Claim for Crown Court Defence | https://claim-crown-court-defence.service.gov.uk/ | Legal Aid Agency project where advocates (solicitors) can submit invoices for work done for legal aid work. project formerly known as ADPproject formerly known as CBO |
-| CCLF | Crown Court Litigator Fee | | LAA web app that manages litigators fee claims. |
+| CCCD | Claim for Crown Court Defence | https://claim-crown-court-defence.service.gov.uk/ | Legal Aid Agency project where advocates (solicitors) can submit invoices for work done for legal aid work. Project formerly known as ADPproject formerly known as CBO |
+| CCLF | Crown Court Litigator Fee | | LAA web app that manages LGFS (Litigators Gradyated Fee Scheme) claims. |
```
[the remark this answers is not in the document]
Comment (carolinapizatto) [reply README.md:92]: Yes it should @jakemulley Thank you for spotting it :)

File: README.md [hunk 2]
```diff
@@ -130,6 +133,7 @@ A better mechanism to enable people without github accounts to suggest changes w
 | COM | Community Offender Managers | | A role within the National Probation Service |
 | COPs | Contracted Out Prisons | | Privately run establishments |
 | COTS | Commercial off-the-shelf | | software; as opposed to custom build |
+| CP | Common Platform | | Refers to the geographic area a CRC is responsible for |
```
[the remark this answers is not in the document]
Comment (donovan-moj) [reply README.md:136]: Absolutely, Jake -- you got to this comment just before I did 😄 .

File: README.md [hunk 3]
```diff
@@ -210,16 +214,19 @@ A better mechanism to enable people without github accounts to suggest changes w
...
+| EDW | Enterprise Data Warehouse |  | Management Information data warehouse. Holds data from many LAA applications. |
 | EE | | | |
 | EM | Electronic Monitoring | | (Also known as 'tagging') is the practice of attaching a tag to someone's ankle, used in England and Wales to monitor curfews and conditions of a court or prison order. |
 | EMI | Electronic Management Information | | LAA web app that allows LAA to share MI data with providers. Currently used for financial statement. |
 | EPICK | Event, Problem, Incident, Continual Service Improvement and Knowledge Team |  |  |
 | EQUIP | Excellence and Quality in Probation Processes | | |
+| ERIC | Early Removal Scheme |  |  |
 | ERP | Enterprise Resource Planning | | |
-| ERS | Early Removal Scheme |  |  |
+| ERS |  |  |  |
```
[the remark this answers is not in the document]
Comment (carolinapizatto) [reply README.md:229]: I think it's ok! I included some historical LAA acronyms in there too @jakemulley

**Comment by carolinapizatto** [comment]
@jakemulley @donovan-moj I made the changes you requested. Let me know if there is anything else you think I should amend! Thank you

**Comment by jakemulley** [comment]
✅ Thanks both!

**Review by jakemulley — APPROVED** [review]

**Pull request merged**